### PR TITLE
Initial port of concentricity project

### PR DESCRIPTION
**Pull request by cjwilliams**
An initial attempt to port the existing concentricity watchface over to Rocky.js

### CODE REVIEW

File: src/rocky/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright [2016] Pebble Technology
+var rocky = require('rocky');
+
+var round = require('./round');
+var rect = require('./rect');
+
+function DrawCommand(platform) {
```
Comment (orviwan) [inline src/rocky/index.js:7]: Perhaps you could just require the relevant file based on platform. 

`if(rocky.watchInfo.platform === 'chalk') {
//require round
} else {
  //require rect
}`

File: src/rocky/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  if (['aplite', 'basalt', 'diorite', 'emery'].indexOf(platform) >= 0) {
+    this.hours = rect.drawHours;
+    this.minutes = rect.drawMinutes;
+    this.seconds = rect.drawSeconds;
+  } else if (platform === 'chalk') {
+    this.hours = round.drawHours;
+    this.minutes = round.drawMinutes;
+    this.seconds = round.drawSeconds;
+  }
+  if (['basalt', 'chalk', 'emery'].indexOf(platform) >= 0) {
```
Comment (orviwan) [inline src/rocky/index.js]: again here, it would be simpler to:
`if (platform === 'diorite') { //bw } else { //color }`

**Review by orviwan — APPROVED** [review]
Looks great, just some optional alternatives.

File: src/rocky/index.js (hunk 1 above)
Comment (cjwilliams) [reply src/rocky/index.js:7]: The require is a directive to webpack to include a module in the bundled JS file. At bundling time, `rocky.watchInfo.platform` doesn't exist (it's an API accessible at runtime) and I'm not sure how webpack would handle this (maybe ignore it?)

File: src/rocky/index.js (hunk 2 above)
Comment (cjwilliams) [reply src/rocky/index.js]: I wish we had a color/bw attribute on watchinfo =(

File: src/rocky/round.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  var centerX = canvasWidth / 2;
+  var centerY = ctx.canvas.unobstructedHeight / 2;
+
+  var strokeWidth = utils.getStrokeWidth(canvasWidth);
+  var outerRadius = utils.calculateOuterRadius(arcId, canvasWidth);
+
+  // Set the drawing color
+  ctx.fillStyle = color;
+
+  if (endAngle == 0) {
```
Comment (orviwan) [inline src/rocky/round.js]: `if (endAngle === 0) {`

**Comment by orviwan** [comment]
One minor comment, other than that LGTM! 👍

**Pull request merged**